docs/unix/quickref.rst

```diff
@@ -0,0 +1,81 @@
+.. _unix_quickref:
+
+Quick reference for the UNIX port
+=================================
+
+Command line options
+--------------------
+
+Usage::
+
+    micropython [ -c <command> | -m <module> | <script> ] [-i ] [ -O<level> ] [ -v ] [ -X <option> ]
```

Comment:
The order here is not correct, the options must precede any filename (<script>).
Here what `micropython -h` outputs for me:
```
usage: ../micropython/ports/unix/micropython [<opts>] [-X <implopt>] [-c <command>] [<filename>]
Options:
-v : verbose (trace various operations); can be multiple
-O[N] : apply bytecode optimizations of level N

Implementation specific options (-X):
  compile-only                 -- parse and compile only
  emit={bytecode,native,viper} -- set the default code emitter
  heapsize=<n>[w][K|M] -- set the heap size for the GC (default 2097152)
```
That's not correct either (-m missing) plus incorrect ordering implication.

Reply:
Thanks for having a look. I have fixed the order.